docs/Contributing/Fleet-UI-Testing.md

```diff
@@ -0,0 +1,362 @@
+# Frontend Testing Strategy & Plan
+
+This document contains the testing strategy and plan for the frontend codebase. The testing
+strategy is a high level overview of the **Who**, **What**, **When** and **Why** when it comes to testing.
+The testing plan primarily outlines the **How** of testing and covers the different practices and
+toolings used in testing.
+
```

Comment:
Include:

For instructions on using our testings tools, check out [/docs/Contributing/Testing-and-local-development.md](./Testing-and-local-development.md)